SpatialGDK/Documentation/content/dynamic-typebindings.md

```diff
@@ -0,0 +1,31 @@
+# Dynamic Typebindings
+To allow Unreal to replicate through the SpatialOS network stack and combine multiple dedicated server instances across one seamless game world, we needed to adapt Unreal networking functionality to work with the [SpatialOS Worker API (SpatialOS documentation)](https://docs.improbable.io/reference/latest/capi/introduction). We did this seamlessly using `Dynamic Typebindings`. `Dynamic Typebindings` operate at runtime so that your iteration speed is not affected despite your network code running on a completely different representation than Unreal’s.
```

Comment:
Using the first person plural is different to all the other docs so, the simplest possible fix is to talk about the GDK doing this (also a few other bits and bobs....

"To allow Unreal to replicate through the SpatialOS network stack and combine multiple dedicated server instances across one seamless game world, The GDK has to adapt Unreal networking functionality to work with the [SpatialOS Worker API (SpatialOS documentation)](https://docs.improbable.io/reference/latest/capi/introduction). It does this seamlessly using `Dynamic Typebindings`. `Dynamic Typebindings` operate at [runtime]({{urlRoot}}/content/glossary#spatialos-runtime) so that your development iteration speed is not affected, despite your network code running on a completely different representation to Unreal’s."